examples/integrate-with-react/src/App.js

```diff
@@ -0,0 +1,33 @@
+import { TypeScriptGenerator } from '../../../';
+import React from 'react';
+import { useState, useEffect } from 'react';
+
+export default function App() {
+  const [models, setModels] = useState('');
+
+  useEffect(() => {
+    (async () => {
+      const generator = new TypeScriptGenerator();
+      const input = {
+        $schema: 'http://json-schema.org/draft-07/schema#',
+        type: 'object',
+        additionalProperties: false,
+        properties: {
+          email: {
+            type: 'string',
+            format: 'email'
+          }
+        }
+      };
+      const models = await generator.generate(input);
+      const modelsCode = models.map((model) => model.result).join('\n');
+      setModels(modelsCode);
+    })();
+  }, []);
```

Comment:
TBH, you don't need to evaluate that code inside component context, you can even create given models outside the component and create them before e.g. rendering main app, but it's only comment from me.

Reply:
You are right yea I just wanted to show how everything could be done within a single function to be completely dynamic 🙂